vignettes/SEIRDV.Rmd

```diff
@@ -0,0 +1,215 @@
+recovered, infectious, recovered and vaccinated individuals. Susceptible
+individuals ($S$) have not been exposed to the virus. Exposed individuals ($E$) have been
+exposed to the virus and have been infected but are not yet infectious to others; they are 
+in the incubation period. Infectious individuals ($I$) can spread the virus to 
+others.  Recovered individuals ($R$) are no longer
+infectious, but they can return to the susceptible population when they lose their
+immunity. Vaccinated individuals ($R$) cannot become infected and are not ill at
+the time, but as it was the case with the recovered population, their immunity
+is temporarily and can become susceptible again with a certain rate. Births and
+deaths unrelated to the infection are not considered in this model. The ODEs describing
+how the five populations evolve over time are
+$$\frac{\text{d}S}{\text{d}t} = -\beta S I - \nu S + \delta_V V + \delta_R R,$$
+$$\frac{\text{d}E}{\text{d}t} = \beta S I -\kappa E,$$
+$$\frac{\text{d}I}{\text{d}t} = \kappa E - (\gamma + \mu) I,$$
+$$\frac{\text{d}R}{\text{d}t} = \gamma I - \delta_R R,$$
+$$\frac{\text{d}V}{\text{d}t} = \nu S - \delta_V V$$
+where $\beta$, $\kappa$, $\gamma$, $\mu$, $\nu$, $\delta_V$ and $\delta_R$ are
+positive parameters. The rate at which susceptible individuals are infected
+depends on the fraction of the population that is susceptible, the fraction of
+the population that is infectious, and the probability of an infectious individual
+infecting a susceptible individual, $\beta$. This is represented in the
+first terms on the right-hand side of $\text{d}S/ \text{d}t$ and $\text{d}E/ \text{d}t$.
+Exposed individuals become infectious at rate $\kappa$. $1/ \kappa$ is
+the incubation time of the virus, i.e. how many days after exposure the individual
+becomes infectious to others. Infectious individuals can either recover or die
+because of the disease. Infectious individuals recover in $1/ \gamma$
+days, thus individuals recover at rate $\gamma$. Similarly, infectious
+individuals die due to the infection after $1/ \mu$ days, making the death rate $\mu$.
+Individuals are vaccinated at rate $\nu$. Among those who are immune (
+recovered and vaccinated), people lose their immunity against the virus at different
+rates depending on what type of immunity they acquired: $\delta_V$ for those vaccinated
+and $\delta_R$ respectively for those previously infected.
+
+In addition to these four main ODEs, the model also keeps track of the cumulative number 
+of cases ($C$) and the cumulative number of disease-related deaths ($D$) over time
+$$\frac{\text{d}C}{\text{d}t} = \beta S I,$$
+$$\frac{\text{d}D}{\text{d}t} = \mu I.$$
+
+The system is closed by specifying the initial conditions
+$$S(0) = S_0,\ E(0) = E_0,\ I(0) = I_0,\ R(0) = R_0,\ V(0) = V_0,\ C(0) = C_0,\ D(0) = D_0,$$
+where $C_0 = E_0 + I_0 + R_0$ to ensure the cases that occur before $t=0$ are
```

Comment:
While this sounds sensible, it doesn't seem to match what we are doing in the code, which is always C_0=0

Reply:
I can just take that piece out to be honest, as we never quite give C0 and D0